src/simmate/apps/badelf/models/badelf.py

```diff
@@ -130,3 +142,58 @@ def update_from_directory(self, directory):
         update_from_directory method here.
         """
         pass
+
+    def update_ionic_radii(self, radii: pd.DataFrame):
+        # pull all the data together and save it to the database. We
+        # are saving this to an ElfIonicRadii datatable. To access this
+        # model, we look need to use "ionic_radii.model".
+        radius_model = self.ionic_radii.model
+        # Let's iterate through the ionic radii and save these to the database.
+        for number, row in radii.iterrows():
+            site_index = row["site_index"]
+            neigh_index = row["neigh_index"]
+            radius = row["radius"]
+            new_row = radius_model(
+                site_index=site_index,
+                neigh_index=neigh_index,
+                radius=radius,
+                bad_elf=self,  # links to this badelf calc
+            )
+            new_row.save()
+
+
+class ElfIonicRadii(DatabaseTable):
+    """
+    This table contains the elf ionic radii calculated during a badelf calculation
+    """
+
+    # class Meta:
+    #     app_label = "workflows"
+
+    site_index = table_column.IntegerField()
+    """
+    The index of the central atom that the radius is for
+    """
+    neigh_index = table_column.IntegerField()
+    """
+    The index of the neighboring atom
+    """
+    radius = table_column.FloatField()
+    """
+    The ELF ionic radius between the central atom and neighbor atom
+    """
+
+    """ Relationships """
+    # each of these will map to a BadELF calculation, so you should typically access this
+    # data through that class.
+
+    # All radii in this table come from a BadELF calculation. There will be
+    # many ionic radii linked to a single calculation and they will all be
+    # stored together here.
+    # Therefore, there's just a simple column stating which badelf calc it
+    # belongs to.
+    bad_elf = table_column.ForeignKey(
```

Comment:
cool! Glad you're testing out foreign keys. Maybe you'll end up like me over time, where I realized JSON columns were easy enough in many cases